style.css

```diff
@@ -372,71 +372,68 @@ h4 {
   background-color: #def0ff;
 }
 
```

Comment:
As required by the previous reviewer, it is important that you follow the design template as closely as possible. This is because in future positions, you'll be required to convert a design into code and the product manager would want you to be detailed as possible. This exercise is to prepare for such situations.
- [ ] The text contents in the card seems too centred, kindly align it to the left as seen in the Figma design.
- [ ] Kindly remove the extra `<` character.
- [ ] There happens to be too much spacing between the elements. Kindly fix that.

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/200107440-197b5fbb-cd1b-443b-be61-c15db777d020.png) | ![image](https://user-images.githubusercontent.com/29071984/200107386-bcedd23d-b1f7-483b-bafb-149ab913c02b.png)

- [ ] After removing the padding as requested by the previous reviewer, Kindly give them a border-radius as seen on the Figma

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/200108025-8e83ee7d-7a38-40e6-a44c-012d39d46bb4.png) | ![image](https://user-images.githubusercontent.com/29071984/200108050-f7a04bfa-2d3c-43b4-9b6c-c28d387fb4b4.png)

- [ ] The submit button now has an underline which wasn't the case before. Kindly fix that.

Your Design | Figma Design
--|--
![image](https://user-images.githubusercontent.com/29071984/200108173-5f1fd129-4f20-4068-adcf-7bcc225d44c1.png) | ![image](https://user-images.githubusercontent.com/29071984/200108146-fdcf0a73-bd30-4b76-80fe-9450e29d9875.png)

- [ ] Lastly, some of the work cards are missing images. I recommend that you fix that.
![image](https://user-images.githubusercontent.com/29071984/200109172-4a720613-8f3d-4a89-87f4-30caf3ca06f3.png)

Reply:
@cwaku @jaflih  @mwanawabangona  @Favourezeugwa thank you I followed your approach and I manage to fix the items on the left and the images. I cannot display them on the dashboard. you can verify.

Reply:
@Carl2678 It's good to know you fixed the issues, well done. 
Also, what do you mean by you can display on the dashboard, can you please elaborate?